feature-libs/user/profile/components/update-password/update-password-component.service.spec.ts

```diff
@@ -166,14 +169,41 @@ describe('UpdatePasswordComponentService', () => {
     });
 
     describe('error', () => {
-      it('should not update the password', () => {
+      it('should not update the password when old password is empty', () => {
         newPassword.setValue('testpassword123');
         service.updatePassword();
         expect(userPasswordFacade.update).not.toHaveBeenCalled();
         expect(globalMessageService.add).not.toHaveBeenCalled();
         expect(routingService.go).not.toHaveBeenCalled();
         expect(authService.coreLogout).not.toHaveBeenCalled();
       });
+      it('should get access denied error message text with agent modifying customer password when feature level is above 6.3', () => {
```

Comment:
I think adding a version description is unnecessary, could you please delete it? `when feature level is above 6.3 `